### PR TITLE
Update README.md with setup info

### LINKED ISSUE #9: Update README.md with setup info
... for local/prod env. necessary configurations (tokens etc)

### PR DESCRIPTION
**Pull request by xjusko**
resolves #9

### CODE REVIEW

File: README.md
Comment (The-Huginn) [inline README.md:10]:
```suggestion
Fork this repository to your GitHub account.
```

File: README.md
Comment (The-Huginn) [inline README.md:35]: I believe this is not yet reflected in main (my fault sorry), but this will most likely end up being an email address instead. We should just keep this in mind and I will try to keep it updated and suggest the change in #40 .

File: README.md
Comment (The-Huginn) [inline README.md:59]:
```suggestion
1. Change the value of `jira-issue-lottery.config-file-repo` in the `application.properties` file to point to a public repository with a valid config file according to first step in [Configuration section](#configure-jira-lottery) such as `https://github.com/jboss-set/jira-issue-lottery`.
```

**Review by The-Huginn — CHANGES_REQUESTED** [review]

File: README.md
Comment (xjusko) [reply README.md:35]: Should I wait until #40 is merged and then edit this PR, or should I change it to an email address now?

Comment (xstefank) [reply README.md:35]: @The-Huginn can fix this in #40

**Review by xstefank — APPROVED** [review]

**Pull request merged**
